rpp-charter.md

```diff
@@ -1,50 +1,34 @@
-Though EPP is still serving the domain name industry well, the progress in available development, integration and operational patterns, tools and technologies create a desire to have a provisioning protocol using the [REST](https://ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm) architectural style and the [JSON](https://datatracker.ietf.org/doc/html/rfc8259) data-interchange format. 
-Such design is expected to take advantage of stateless architecture and widely deployed solutions such as OpenAPI, associated documentation, testing and code generation tools, and L4-L7 network services such as API gateways, authorization servers, load balancers, web servers, WAFs etc. 
-The successful adoption of RDAP by Domain Name Registries (DNRs) and Regional Internet Registries (RIRs) demonstrates the usefulness of this type of architecture and technology tool stack.
+Though EPP is still serving the domain name industry well, the progress in available development, integration and operational patterns, tools and technologies create a desire to have a provisioning protocol using the REST architectural style and the JSON data-interchange format. 
+Such design is expected to take advantage of stateless architecture and widely deployed solutions such as OpenAPI, associated documentation, testing and code generation tools, and L4-L7 network services such as API gateways, authorization servers, load balancers, web servers, web application firewalls, etc. 
+The successful adoption of Registration Data Access Protocol (RDAP) by Domain Name Registries (DNRs) and Regional Internet Registries (RIRs) demonstrates the usefulness of JSON and REST based architectures.
 
-This evolution has already started with some Country Code Top Level Domains (ccTLDs). 
-First production deployments of this approach have seen adoption by both new and existing clients, where a preference has been noted for this approach.
-A REST architecture is expected to allow easier integration between registries and registrars, thus lowering the costs for domain registration and new market entrants.
+Production deployments of REST and JSON based provisioning protocols by Country Code Top Level Domains (ccTLDs) have been well received, however variance across implementations is starting to become a concern.
+
+A standardized REST architecture is expected to allow easier integration between registries and registrars, thus lowering the costs for domain registration and new market entrants.
 
 # Scope
 
 The RPP WG is tasked with creating a series of specifications to be known collectively as the RESTful Provisioning Protocol (RPP).
-These specifications target a REST architecture high on the Richardson Maturity Model using HTTPS and JSON. 
+These specifications target at least level 2 of the Richardson Maturity Model, using HTTPS and JSON. 
```

Comment:
in all the discussions so far the open point was whether to use hypermedia controls or not.
The use of HTTP verbs was never put under doubt and was in all the initial drafts from @mwullink 

This would be level 3 as minimum.

Reply:
Level 2 is HTTP verbs if we start from level 0, right?

Reply:
https://martinfowler.com/articles/richardsonMaturityModel.html

https://restfulapi.net/richardson-maturity-model/

At least level 2 means, HATEOAS is not mandatory, but multiple URIs and verbs are.

Reply:
Of course. Sorry for confusion. Level 2 is then correct.